lib/ZnapZend.pm

```diff
@@ -129,7 +129,7 @@ my $refreshBackupPlans = sub {
     $self->backupSets($self->zConfig->getBackupSetEnabled($dataSet));
 
     @{$self->backupSets}
-        or $self->zLog->warn("No backup set defined or enabled, yet. run 'znapzendzetup' to setup znapzend");
+        or die "No backup set defined or enabled, yet. run 'znapzendzetup' to setup znapzend";
```

Comment:
can you add a .. \n" at the end of your string so that the error message looks nice in the output.

Reply:
I think I did.